docs/guides/staging-patches.md

```diff
@@ -0,0 +1,122 @@
+
+## Prerequisites
+
+This guide assumes the Shorebird command-line is installed on your machine and that you are logged into an account. Refer to the [getting started](/) instructions for more information.
+
+## Create a Project
+
+:::info
+In this guide, we'll go through the process of creating a new project from scratch. To apply these changes to an existing project, skip this step and read ahead.
+:::
+
+Create a new project using `flutter create example --empty`.
+
+## Initialize Shorebird
+
+Next, initialize Shorebird in the current project via `shorebird init`.
+
+```
+shorebird init
+```
+
+The generated `shorebird.yaml` should look something like:
+
+```yaml
+# Your app_id is not a secret and is just used to identify your app
+# when requesting patches from Shorebird's servers.
+app_id: ee322dc4-3dc2-4324-90a9-04c40a62ae76
+# auto_update controls if Shorebird should automatically update in the background on launch.
+# If auto_update: false, you will need to use package:shorebird_code_push to trigger updates.
+# https://pub.dev/packages/shorebird_code_push
+# Uncomment the following line to disable automatic updates.
+# auto_update: false
+```
+
+## Create a release
+
+Now that we've created our apps on shorebird, we need to create releases (one for each platform). To create a release, we'll use the `shorebird release` command.
+
+```sh
+# Create a release for Android
+shorebird release android
+
+# Create a release for iOS
+shorebird release ios-alpha
+```
+
+We can verify the releases were created successfully by visiting [Shorebird console](https://console.shorebird.dev).
+
+You should also [submit the generated app bundles to the Play Store](/guides/release/android#upload-to-the-play-store) and [submit the generated ipa to the App Store](/guides/release/ios#upload-to-the-app-store).
+
+## Creating a patch
+
+Now that we have our releases on the Play Store and App Store, we can create a patch using `shorebird patch`. For the sake of this example, let's set the `backgroundColor` of the `Scaffold` to `Colors.cyan` in `lib/main.dart`:
+
+```diff
+import 'package:flutter/material.dart';
+
+void main() {
+  runApp(const MainApp());
+}
+
+class MainApp extends StatelessWidget {
+  const MainApp({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return const MaterialApp(
+      home: Scaffold(
++       backgroundColor: Colors.cyan,
+        body: Center(
+          child: Text('Hello World!'),
+        ),
+      ),
+    );
+  }
+}
+
+```
+
+Now that we've applied the changes, let's create a patch:
+
+```sh
+# Stage a patch on Android
+shorebird patch android --staging
+
+# Stage a patch on iOS
+shorebird patch ios-alpha --staging
+```
+
+## Preview the release
+
+Next, preview the app release locally on a device or emulator, use `shorebird preview`.
+
+```sh
+# Preview the release in the staging environment
+shorebird preview --staging --app-id ee322dc4-3dc2-4324-90a9-04c40a62ae76 --release-version 1.0.0+1
```

Comment:
why app-id here?  I guess so you can run it outside of the app dir?

Reply:
yeah it was just for the sake of being explicit and since you can run this command from anywhere